spec/features/archiving_tags_for_mainstream_browse_pages_spec.rb

```diff
@@ -0,0 +1,147 @@
+require 'rails_helper'
+require 'gds_api/test_helpers/content_store'
+
+RSpec.feature "Archiving tags for mainstream browse pages" do
```

Comment:
Small thing, but I think this would be clearer if worded "Archiving mainstream browse page tags".

Same with "Archiving topic tags", plus the corresponding filenames.